larray/viewer.py

```diff
@@ -2418,7 +2426,8 @@ def open_documentation(self):
 
     def set_current_file(self, filepath):
         self.update_recent_files([filepath])
-        self.currentFile = filepath
+        self.current_file = filepath
+        # TODO: update window title
 
```

Comment:
this is already the case, but the trick is that set_current_file can be called in cases where set_widget_array is not called (eg on save), and the title should still be update in that case.